companies/pinterest.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Pinterest
+tags: ["tech", "social media"]
+image: NewsroomPost-Header-200602_0.jpg
+blm_statements:
+  - url: https://newsroom.pinterest.com/en/post/listening-and-acting
+    date_posted: 2020-06-02
+    type: Blog post >
+        Hello.
+
+        This morning we hosted an open conversation with our entire company.
+
+        We heard directly from our Black employees about the pain and fear they feel every day living in America. Their first-hand, lived experiences of racism and injustice. How the murder of George Floyd—a friend, a father, a son—forced the world to face the reality that the systemic racism facing the Black community for generations remains very real today. It’s wrong. This has got to change.
+
+        Because people are being murdered. And Black Lives Matter.  
+
+        We want to express not only our solidarity but also our commitment to taking action. Not just now, but from now on. Our team, with input from outside experts, identified steps we believe will make a difference over time.
+
+        On our platform
+
+        We are elevating content on racial justice, such as resources for how parents can talk to their children; organizations users can support; and educational information about systemic racism in America.
+        We are working to make sure the content people see on Pinterest represents people from diverse backgrounds. We are investing more resources into growing the diversity of content on our platform. 
+        We won’t serve ads on Black Lives Matter results, so people can focus on learning about the movement.
+        Within our company
+
+        We will continue to make progress on improving the diversity of our company, with a deeper focus on increasing the diversity of senior leaders. 
+        We’re offering more allyship resources, manager training, ongoing company-wide conversations about race. 
+        To support the efforts of others
+
+        We’re donating 25,000 shares of stock (currently valued at roughly $500,000) to organizations committed to racial justice and promoting tolerance. We’ll share more details on our partnership with these organizations in the coming days.
+        We’re making a $250,000 investment to help rebuild local businesses damaged in the protests.
+        We’re donating $750,000 in paid media to organizations that support racial justice. 
+        This is not just a moment in time. It will require a sustained effort for the lifetime of our company. 
+
+        With everything we do, we will make it clear that our Black employees matter, Black Pinners and creators matter, and Black Lives Matter.  
+
+        Ben
+    raw_text: Reeling from charges of racism and gender discrimination, Pinterest on Monday named a Black executive to its board of directors.
+  - url: https://www.usatoday.com/story/money/2020/08/17/pinterest-adds-black-exec-its-boards-following-racism-allegations/3379478001/
```

Comment:
Nice find!! These URLs that aren't directly from the company can go in the "additional resources" section.  I'll clarify that in our instructions.

Reply:
fixed this!